### ISSUE TITLE
Nuxt fails to resolve plugin when using 'auth' option.

### ISSUE THREAD

**Issue by juni0r** [Tahul/nuxt-edgedb#1]
Nuxt 3.8.1 fails to resolve a plugin when auth is enabled. It's trying to resolve `node_modules/nuxt-edgedb-module/dist/runtime/plugin/edgedb-auth.ts`, but since this is the build output, it's supposed to be `edgedb-auth.mjs`.

[Demo repo with minimal setup here](https://github.com/juni0r/nuxt-edgedb)

> nuxt.config.ts
```ts
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: ['nuxt-edgedb-module'],
  edgeDb: {
    auth: true,
  },
})
```

<img width="879" alt="Screenshot 2023-11-20 at 17 41 41" src="https://github.com/Tahul/nuxt-edgedb/assets/11867/9de80d27-024e-45bb-9835-006a9a5f4851">

**Comment by Tahul** [comment]
Hey @juni0r ; thank you for reporting the issue and for the reproduction.

I just made this [fix](https://github.com/Tahul/nuxt-edgedb/commit/dd92cad6cb4c8a4a77d336a7e06859c5f1315924) and published it on `0.0.8`.

Could you try it and let me know if that solves it?

**Comment by juni0r** [comment]
Hey @Tahul, thanks for the quick fix! Plugin gets loaded now but I ran into another issue. Will open a separate issue.

That's a hell of a module you created there btw. I read the code and learnt a few things. Can't wait to get it to work.
